service/src/main/kotlin/fi/espoo/evaka/attendance/MobileUnitController.kt

```diff
@@ -97,18 +118,84 @@ fun Database.Read.fetchUnitInfo(unitId: DaycareId, date: LocalDate): UnitInfo {
     // language=sql
     val groupsSql =
         """
-        SELECT g.id, g.name
+        WITH child AS (
+            SELECT
+                dgp.daycare_group_id as group_id,
+                SUM(COALESCE(an.capacity_factor, 1) * CASE
+                    WHEN dc.type && array['FAMILY', 'GROUP_FAMILY']::care_types[] THEN 1.75
+                    WHEN extract(YEARS FROM age(ca.date, ch.date_of_birth)) < 3 THEN coalesce(sno.occupancy_coefficient_under_3y, default_sno.occupancy_coefficient_under_3y)
+                    ELSE coalesce(sno.occupancy_coefficient, default_sno.occupancy_coefficient)
+                END) AS capacity
```

Comment:
It's pretty unfortunate that this formula is now duplicated multiple times. Could it be extracted into a function?

Reply:
Discussed on slack and came to the conclusion not to change this as the duplication is only in this file.